hazelcast/proxy/vector_collection.py

```diff
@@ -91,10 +198,28 @@ def search_near_vector(
         *,
         include_value: bool = False,
         include_vectors: bool = False,
-        limit: int,
+        limit: int = 10,
         hints: Dict[str, str] = None
     ) -> Future[List[SearchResult]]:
+        """Returns the Documents closest to the given vector.
+
+        The search is performed using distance metric set when
+        creating the vector index.
+
+        Args:
+            vector: The vector to be used as the reference.
+                It must have the same dimension as specified when creating the vector index.
+            include_value: Return value attached to the Document.
+            include_vectors: Return vectors attached to the Document.
+            limit: Limit the maximum number of Documents returned.
+                If not set, the server default is used.
```

Comment:
```suggestion
            limit: Limit the maximum number of Documents returned.
```

Reply:
Added the default `10` at:
https://github.com/hazelcast/hazelcast-python-
client/pull/678/commits/75ff67f5ca7811e094e2a3603b04e59f51eeceab

Reply:
shouldn't we DRY? default `limit` is clearly visible as default value of the method parameter

Reply:
The defaults are explicitly mentioned in the documentation in some other data structure methods, like `Map.add_index`, so I think it's OK to keep it.